data/scripts/network/blessings_window.lua

```diff
@@ -44,15 +44,17 @@ local function sendBlessingsWindow(player)
 	local equipmentLoss = {100, 70, 45, 25, 10, 0, 0, 0}
 	local equipmentPercent = equipmentLoss[#blessings + 1]
 
+	local playerSkull = player:getSkull()
+	playerSkull = (playerSkull == SKULL_RED or playerSkull == SKULL_BLACK)
```

Comment:
Please don't reuse variables for different meanings. You can use `if playerSkull >= SKULL_RED` below